extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/ServerLimitsConfig.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.vertx.http.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.configuration.MemorySize;
+
+@ConfigGroup
+public class ServerLimitsConfig {
+    /**
+     * The the maximum length of all headers.
+     */
+    @ConfigItem(defaultValue = "8192")
+    public MemorySize maxHeaderSize;
```

Comment:
I am not sure if this is big enough as a default, some sites have a lot of cookies, even for this page on github my browser is sending 1.4k of headers.

Reply:
I'll update the default value to something like `20K`.